src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.scss

```diff
@@ -35,6 +34,24 @@
     z-index: $overlay-z-index;
 }
 
+.bcs-AnnotationActivityMenu {
```

Comment:
Should this live in its component's stylesheet? Or should we pass it a class name like `bcs-AnnotationActivity-menu` to stay consistent with SUIT naming?